contracts/V3Migrator.sol

```diff
@@ -43,7 +51,7 @@ contract V3Migrator is IV3Migrator, Multicall, SelfPermit {
 
     function migrate(MigrateParams calldata params) external override {
         // burn v2 liquidity to this address
-        IUniswapV2Pair(params.pair).transferFrom(msg.sender, params.pair, params.liquidityV2);
+        IUniswapV2Pair(params.pair).transferFrom(msg.sender, params.pair, params.liquidityToMigrate);
         (uint256 amount0V2, uint256 amount1V2) = IUniswapV2Pair(params.pair).burn(address(this));
 
         // get underlying tokens
```

Comment:
probably cheaper to take v2Factory, token0, token1 and compute the pair, than make these two calls (although idk if the bytecode hash is same for all forks, probably not. should we prioritize uniswap gas cost tho?)

Reply:
for interop i prefer explicit pair address. perhaps we could just pass the token addresses in calldata?

Reply:
yeah that works